apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.scss

```diff
@@ -1,3 +1,13 @@
 .closingDateControl {
   margin-top: 1.5rem;
+}
+
+.closingStageContainer {
+  p {
+    max-width: 100%;
+  }
+
+  .reduceVerticalSpacing {
+    margin-top: 0rem;
+  }
 }
```

Comment:
These styles don't appear to be used.  Can we remove them?

Reply:
resolved and removed. Since none of the styles are no longer used, the file is being deleted.